src/main/groovy/com/kevalpatel2106/gradle/jacoco/android/JacocoAndroidPlugin.groovy

```diff
@@ -84,21 +84,21 @@ class JacocoAndroidPlugin implements Plugin<ProjectInternal> {
     }
     reportTask.reports {
       def destination = project.jacocoAndroidUnitTestReport.destination
-      
-      csv.enabled project.jacocoAndroidUnitTestReport.csv.enabled
-      html.enabled project.jacocoAndroidUnitTestReport.html.enabled
-      xml.enabled project.jacocoAndroidUnitTestReport.xml.enabled
+
+      csv.required project.jacocoAndroidUnitTestReport.csv.enabled
```

Comment:
Hello, I am getting a crash on this line

`Caused by: org.codehaus.groovy.runtime.metaclass.MissingMethodExecutionFailed: No signature of method: org.gradle.api.reporting.internal.TaskGeneratedSingleFileReport.required() is applicable for argument types: (Boolean) values: [false]`

It probably should be `csv.required = project.jacocoAndroidUnitTestReport.csv.enabled`